### PR TITLE
Add label and altLabel to the GND label fields

### LINKED ISSUE #354: URI without label shown for GND resource
At least one GND resource makes problems: http://d-nb.info/gnd/5183585-X See https://nwbib.de/search?person=http%3A%2F%2Fd-nb.info%2Fgnd%2F5183585-X.

### PR DESCRIPTION
**Pull request by fsteeg**
Will resolve #354

These are new fields used in the Lobid 2.0 data.

### CODE REVIEW

**Comment by fsteeg** [comment]
Deployed to staging, see:

https://test.nwbib.de/HT019093814
https://test.nwbib.de/HT017324477

Two remarks:

In the first example, the two contributors have the same label. This seems to be an error in the Lobid data 2.0 conversion. Compare `200` and `204` in http://lobid.org/hbz01/HT019093814 and `contributor` in http://lobid.org/resources/HT019093814.

In the second example, the `altLabel` is used, since the `label` is an array, see http://lobid.org/resources/HT017324477. Multiple labels are not expected by the nwbib UI at this point. Are these on purpose in the Lobid 2.0 data?

**Comment by fsteeg** [comment]
@acka47: Meta remark: trying functional review in the pull request as you suggested. Finding it odd to mix the levels though. Pull request is about adding two fields in some code part, 'deployed to staging' message is about things that are not described here, but in the issue. Let's see how this works in practice.

**Comment by acka47** [comment]
> In the first example, the two contributors have the same label. This seems to be an error in the Lobid data 2.0 conversion. Compare `200` and `204` in http://lobid.org/hbz01/HT019093814 and `contributor` in http://lobid.org/resources/HT019093814.

I created https://github.com/hbz/lobid-resources/issues/128 for this.

> In the second example, the `altLabel` is used, since the `label` is an array, see http://lobid.org/resources/HT017324477. Multiple labels are not expected by the nwbib UI at this point. Are these on purpose in the Lobid 2.0 data?

Created https://github.com/hbz/lobid-resources/issues/129 to deal with this.

**Comment by acka47** [comment]
+1

**Comment by dr0i** [comment]
+1

File: app/views/tags/result_doc.scala.html [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 @gndEntities = @{ 
 	Option(Seq(
+	    "label",
+	    "altLabel",
 	    "preferredName",
 	    "preferredNameForTheWork",
 	    "preferredNameForTheFamily",
```
Comment (dr0i) [inline app/views/tags/result_doc.scala.html]: We don't use the `preferred*` anymore in Data 2.0.

Comment (fsteeg) [inline app/views/tags/result_doc.scala.html]: We still have a fallback to use API 1.x, will remove these with other code after the 2.0 launch.

**Pull request merged**